src/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,200 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
+    Copyright (C) 2019 Stiftung Pillar Project
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+import * as React from 'react';
+import { View, Animated } from 'react-native';
+
+import styled from 'styled-components/native';
+
+import { Paragraph } from 'components/Typography';
+import { spacing, fontStyles } from 'utils/variables';
+import { themedColors } from 'utils/themes';
+import { getDeviceWidth, reportLog } from 'utils/common';
+import { measure } from 'utils/ui';
+
+
+const screenWidth = getDeviceWidth();
+const ARROW_SIZE = 10;
+const TOOLTIP_WIDTH = 230;
+const SIDE_MARGIN = 20;
+
+type Layout = {
+  x: number,
+  y: number,
+  width: number,
+  height: number,
+};
+
+type Props = {
+  body: string,
+  positionOnBottom?: boolean,
+  isVisible: boolean,
+  wrapperStyle?: Object,
+  children?: React.Node,
+};
+
+const TooltipParagraph = styled(Paragraph)`
+  color: ${themedColors.control};
+  ${fontStyles.regular}px;
```

Comment:
Looks like those styles are from the walkthrough tooltip, but Isn't `fontStyles.regular` an object?
```suggestion
  ${fontStyles.regular};
```